### DISCUSSION TITLE
Target #101: Gradients

### DISCUSSION THREAD

**Discussion by Gabb192** [cssbattle/forum#240] [Q&A] [answered]
Hey there,

Been trying to improve my high score on [target #101](https://cssbattle.dev/play/101) with this solution:

`…radial-gradient(265q,#A22015 14%,#000 0 20%,#0000 0)…`

The problem is that i have to repeat the same gradient 3x times for the circles, is there a way to write that only once (maybe with var(?))?

**Comment by APerricone** [comment, marked as answer]
sure you can use var, for example:
```--r:radial-gradient(265q,#A22015 14%,#000 0 20%,#0000 0)``

**Reply by Gabb192** [reply]
Thank you a lot! I didn’t know that👍

**Reply by gunnarbittersmann** [reply]
It should be shorter to use the same declaration for multiple elements:  
`foo,bar,baz{radial-gradient(265q,#A22015 14%,#000 0 20%,#0000 0)`

**Reply by Gabb192** [reply]
@gunnarbittersmann It works, thanks!

**Comment by himanshubhatt1** [comment]
<style>*{--a:radial-gradient(1q,#0000,69q,#243D83 0 90q,#0000);background:var(--a)-80q 0,var(--a)80q 0q,var(--a)0 80q,var(--a)0-79q,radial-gradient(1q,#6592CF,25vw,#243D83


 In target #96 why i got only 99.9%......

**Reply by Gabb192** [reply]
Somethimes the problems are caused just for a pixel... I don't really like to share here the solutions of your 99.9%, where can i contact you? So im gonna send you that @himanshubhatt1 . If it's a problem, im gonna share that here!

**Reply by Gabb192** [reply]
done! 😃
